request_test.cc

```diff
@@ -0,0 +1,138 @@
+#include "gtest/gtest.h"
+#include "request.h"
+#include <string>
+
+class RequestTest : public ::testing::Test {
+  protected:
+    std::string CreateRequestString(std::string method, std::string uri, 
+                            std::string version)
+    {
```

Comment:
Can you include the opening curly brace on the same line as the function declaration. Same thing for the other functions in this file.
Example
void func() {

}